app/src/main/java/com/firebase/uidemo/database/ChatActivity.java

```diff
@@ -70,8 +75,10 @@ protected void onCreate(Bundle savedInstanceState) {
         mManager.setReverseLayout(false);
 
         mMessages = (RecyclerView) findViewById(R.id.messagesList);
-        mMessages.setHasFixedSize(false);
+        mMessages.setHasFixedSize(true);
```

Comment:
Kinda unrelated to the point of this PR but I feel like we keep flipping this boolean back and forth, what did this change do?